Main Data Classes -- Task#2

TatyanaGudym opened the pull request:
1. There is a string. Find a maximum number of digits placed in a row in it

TatyanaGudym committed:
Main Data Classes -- Task#2

2. There is a string. Find a maximum number of digits placed in a row in it

d-konovalov commented on Main_Data_Class_2.rb:
```diff
@@ -0,0 +1,7 @@
+# 2. There is a string. Find a maximum number of digits placed in a row in it
```
'Find maximum numbers of digits placed in a row' means that we shouldn't find a maximum number in a string but we should find a maximum number of chars placed in a row where each is a digit. For example:
the string is 'lkj5d54d454d5454654d654d6546'
your method should return '6' as an answer since we have here few groups of digits and fourth is the biggest one

TatyanaGudym committed:
New commit #2 for Main Data Classes Task2

Method is updated. Please take a look and advise if I can do it simpler?
Thank you.

TatyanaGudym commented:
Hi Dima,
I have updated the method. Please take  look and advise.
Thank you!
